qemu/tests/blockdev_commit_hotunplug.py

```diff
@@ -0,0 +1,58 @@
+from virttest import utils_misc
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitHotunplug(BlockDevCommitTest):
+
+    def check_status(self, device):
+        if device not in str(self.main_vm.monitor.info_block()):
```

Comment:
what about
1. return device not in str(self.main_vm.monitor.info_block())
2. check_status -> is_device_deleted?

Reply:
Ok,thanks a lot.